src/commands/variations/__snapshots__/create.test.ts.snap

```diff
@@ -74,7 +74,9 @@ exports[`variations create creates a variation and returns the full feature in i
 `;
 
 exports[`variations create errors when missing required flags in headless mode 1`] = `
-"key is a required field
+"(node:26936) Warning: Mismatched version in @devcycle/cli plugin manifest. Expected: 5.19.0 Received: 5.18.0
```

Comment:
bug / bad commit? 🤔

Reply:
dunno this is what happened when I ran tests